src/views/Swap/components/ImportTokenWarningModal.tsx

```diff
@@ -8,7 +8,7 @@ interface Props extends InjectedModalProps {
   onCancel: () => void
 }
 
-const TokenWarningModal: React.FC<Props> = ({ tokens, onDismiss, onCancel }) => {
+const ImportTokenWarningModal: React.FC<Props> = ({ tokens, onDismiss, onCancel }) => {
```

Comment:
Rename `TokenWarningModal` to distinguish it more clearly from the new `SwapWarningModal`